specs/saltpack_signing_v2.md

```diff
@@ -16,7 +16,7 @@ properties on top of a standard NaCl signature:
 
 We define two signing formats: one attached and one detached. The attached
 format will have a header packet and payload packets similar to the encryption
-format, with a final packet flag at the end of the message, and an incrementing
```

Comment:
The title suggests we're removing the reference to an empty payload packet, but here it looks like we're removing the reference to the final packet flag. Did you commit the wrong thing?

Reply:
Yes, sorry!